web/src/components/tasks/business-formation/BusinessFormationPaginator.test.tsx

```diff
@@ -582,6 +583,32 @@ describe("<BusinessFormationPaginator />", () => {
         expect(screen.getByRole("alert")).toHaveTextContent(Config.formation.fields.trustees.label);
       });
 
+      it.each(["limited-liability-company", "limited-liability-partnership"])(
+        "shows the signature is missing error text in the alert when legalStructure is %s",
```

Comment:
Can we update the description so that's it's clear? Something like: `"shows an error text when the signature is missing'`?